docs/node/run-your-node/paratime-client-node.mdx

```diff
@@ -216,6 +216,58 @@ accessible on the internet (for `TCP` and `UDP` traffic).
 
 :::
 
+## Configuring TEE Paratime Client Node
+
+If your node requires the ability to issue queries that can access confidential
+data, start by following the [Configuration](#configuration) section to create
+the `/node/etc/config.yml` file. Once the file is set up, add the following
+content to the `runtime` part within the `/node/etc/config.yml` file:
+
+```yaml
+# ... sections not relevant are omitted ...
+runtime:
+    # Paths to ParaTime bundles for all of the supported ParaTimes.
+    paths:
+        - {{ runtime_orc_path }}
+    environment: sgx
+    config:
+      {{ paratime_id }}:
+          estimate_gas_by_simulating_contracts: true
```

Comment:
This also needs to be moved below, it goes hand-in-hand with expensive queries and is unrelated to TEE client nodes.